arviz/rcparams.py

```diff
@@ -153,23 +153,40 @@ def validate_with_none(value):
 
 def _validate_bokeh_marker(value):
     """Validate the markers."""
-    all_markers = (
-        "Asterisk",
-        "Circle",
-        "CircleCross",
-        "CircleX",
-        "Cross",
-        "Dash",
-        "Diamond",
-        "DiamondCross",
-        "Hex",
-        "InvertedTriangle",
-        "Square",
-        "SquareCross",
-        "SquareX",
-        "Triangle",
-        "X",
-    )
+    try:
+        from bokeh.core.enums import MarkerType
+    except ImportError:
+        MarkerType = [
+            "asterisk",
```

Comment:
isn't this a breaking change? going from CamelCase to snake_case?

Reply:
Should we add both cases?

I think if bokeh is installed it will use the library part

Reply:
I think we can have a breaking change here and it is probably not worth it to try and cover both options.